packages/vega-scenegraph/test/svg-renderer-test.js

```diff
@@ -188,7 +188,7 @@ tape('SVGRenderer should support enter-item redraw', t => {
   rect2.bounds = new Bounds().set(70, 10, 120, 60);
   rects.items.push(rect2);
 
-  var svg = compensate(r.render(scene, [rect1, rect2]).svg());
+  var svg = compensate(r.render(scene).svg());
```

Comment:
Then `render` method previously had a single argument (the scene) so the second argument didn't do anything. Now the second argument is the optional array of mark types, so I removed the second argument in these tests